content/05.proteolysis.md

```diff
@@ -8,13 +8,59 @@ Peptides that are too short are less likely to uniquely match to a single protei
-Trypsin is the most common choice of protease for proteome hydrolysis [@DOI:10.1002/mas.21376]. 
- 
+Trypsin is the most common choice of protease for proteome hydrolysis [@DOI:10.1002/mas.21376].
+Trypsin is favorable because of its specificity, availability, efficiency and low cost. 
+Trypsin cleaves at the C-terminus of basic amino acids, arginine and lysine. 
+Many of the peptides generated from trypsin are short in length (less than ~ 20 amino acids), which is ideal for chromatographic separation, MS-based peptide fragmentation and identification by database search.
+The main drawback of trypsin is that majority (56%) of the tryptic peptides are ≤ 6 amino acids, and hence using trypsin alone limits the observable proteome [@PMID: 20113005; @PMID: 25823410; @PMID:30687733].
+This limits the number of identifiable protein isoforms and post translational modifications.
+
 
 3. theoretical studies of proteolysis enzymes [@DOI:https://doi.org/10.1155/2014/960902]
 4. Challenges associated with alternative enzyme choices (non-specific and semi-specific enzymes)
-5. Alternative enzyme choices (one paragraph each?) - LysC
-6. GluC
-7. AspN
-8. Alpha-lytic protease [@DOI:https://doi.org/10.1074/mcp.m113.034710] and how it enables mapping human SUMO sites [@PMID:29079793].
-9. others?
+
+Many alternative proteases are available with different specificities that complement trypsin to reveal different proteomic sequences [@PMID:12643544; @PMID:20113005], which can help distinguish protein isoforms [@PMID:27123950].
+The enzyme choice mostly depends on the application.
+In general, for a mere protein identification mostly trypsin is the choice due to the reasons aforementioned.
+However, alternative enzymes are used for de novo assembly when the genomic data information is limited in the public database repositories.
+Use of multiple proteases for proteome digestion also can improve the sensitivity and accuracy of protein quantification [@PMID:30336047].
+Lysyl endopeptidase (Lys-C) obtained from  _Lysobacter enzymogenesis_ is a serine protease invloved in cleaving carboxyl terminus of lysine [@PMID:6359954, @PMID:25823410].
+Like trypsin, the optimum pH required for its activity is from pH 7-9. 
+One among the major advantage of Lys-C is that it is resistant to denaturing agents, including 8 M urea [@PMID: 27123950].
+Compared to Arginine, trypsin is less efficient in cleaving Lysine, which could limit the quality of quantitation from tryptic peptides.
+Hence to achieve complete protein digestion with minimal missed cleavages, Lys-C is often used in parallel to complement tryspin digestion [@PMID:23017020]. 
+Glutamyl peptidase I, commonly known as Glu-C or V8 protease, is a serine protease obtained from _Staphyloccous aureus_ [@PMID: 4627743].
+This specifically cleaves at the C-terimus of Glu. 
+However, depending on the pH conditions, in addition to Glu (pH 4.0) it is invloved in cleaving Asp (pH 8.0) residues also [@PMID:4627743; @PMID:26748652 ].
```

Comment:
the citation you added argues the specificity is the same at ph 4.5 or pH 8, which seems like the opposite of this sentence that seems to argue a different preference at different pH